common/src/main/java/net/minescript/common/Minescript.java

```diff
@@ -2618,23 +2589,59 @@ private static Optional<JsonElement> handleScriptFunction(
           return Optional.of(result);
         }
 
-      case "player_set_position":
-        {
-          args.expectSize(5);
-          double x = args.getDouble(0);
-          double y = args.getDouble(1);
-          double z = args.getDouble(2);
-          float yaw = player.getYRot();
-          float pitch = player.getXRot();
-          if (args.get(3) != null) {
-            yaw = args.getDouble(3).floatValue();
-          }
-          if (args.get(4) != null) {
-            pitch = args.getDouble(4).floatValue();
-          }
-          player.moveTo(x, y, z, yaw, pitch);
-          return OPTIONAL_JSON_TRUE;
+      case "player_set_position": {
+        args.expectSize(5);
+        double x = args.getDouble(0);
+        double y = args.getDouble(1);
+        double z = args.getDouble(2);
+        float yaw = player.getYRot();
+        float pitch = player.getXRot();
+
+        if (args.get(3) != null) {
+          yaw = args.getDouble(3).floatValue();
+        }
+        if (args.get(4) != null) {
+          pitch = args.getDouble(4).floatValue();
         }
+        player.moveTo(x, y, z, yaw, pitch);
+        return OPTIONAL_JSON_TRUE;
+      }
+
+      case "player_set_sprinting":
+      {
+        args.expectSize(1);
+        boolean x = args.getBoolean(0);
+        player.setSprinting(x);
+
+        return OPTIONAL_JSON_TRUE;
+      }
+
+      case "player_add_velocity":
+      {
+        args.expectSize(3);
```

Comment:
There's a newer version of this which allows you to name the args for more informative error messages:

`args.expectArgs("x", "y", "z");`